### DISCUSSION TITLE
Latest version of url-frontier

### DISCUSSION THREAD

**Discussion by wbartussek** [crawler-commons/crawler-commons#431] [Q&A]
Hi there, I would like to use the latest version of url-frontier. But here it seems that further development has stopped. How should I proceed? What is your recommendation?
Regards, Wolfram

**Comment by jnioche** [comment]
Hi @ontoport 

More of a question for  [https://github.com/crawler-commons/url-frontier](https://github.com/crawler-commons/url-frontier/discussions) really.

>  I would like to use the latest version of url-frontier. 

The README in https://github.com/crawler-commons/url-frontier/tree/master/service should get you started

> But here it seems that further development has stopped. How should I proceed? What is your recommendation?

Give it a try, ask questions if something is not clear, see if it does what you expect and if you see things that could be done better, suggest improvements. Contributions are always welcome!
